### DISCUSSION TITLE
Add the option to use an alternative (numeric) code instead of just Touch ID

### DISCUSSION THREAD

**Discussion by PeterTGarner** [isontheline/pro.webssh.net#928] [Ideas]
I am happy with using Touch ID to secure access to WebSSH but if I have to ask a trusted person to access the app I don't want them to have to register a fingerprint. In this case, a (say) 4-6 digit code could be used instead. Thank you!

**Comment by isontheline** [comment]
Hey @PeterTGarner 👋

Thank you so much for your feedback 🙏

I understand your issue and point of view but, if user has already an access to your unlocked device why not just turn off WebSSH Face / Touch ID ?

Or even if you don’t want to add another fingerprint (which I totally understand) why not giving your device password?

Maybe I don’t have understand all your needs about your request. If so could you drive me with use cases?

Again thank you a lot!
